Description: Upstream changes introduced in version 5.22.1-3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 maxima (5.22.1-3) unstable; urgency=low
 .
   * build-dep latest gcl to fix mips/ppc
 .
 The person named in the Author field signed this changelog entry.
Author: Camm Maguire <camm@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- maxima-5.22.1.orig/doc/man/ru/maxima.1
+++ maxima-5.22.1/doc/man/ru/maxima.1
@@ -10,7 +10,7 @@
 .ds Cl "Common L\s-2isp\s+2
 .ds Li "L\s-2isp\s+2
 .ds Ky "GNU Common L\s-2isp\s+2
-.ds Pr /usr/local
+.ds Pr /usr
 .ds Vr 5.22.1
 
 .SH Õ¿«¬¿Õ»≈
